packages/react-components/src/components/Drawer/DrawerMenuItem.tsx

```diff
@@ -54,6 +56,12 @@ export interface _DrawerMenuItemProps {
    * Property to set the focus since this is the first item in the menu
    */
   shouldFocusOnMount?: boolean;
+  /* @conditional-compile-remove(reaction) */
+  reactionResources?: ReactionResources;
```

Comment:
can remove these now

Reply:
No. Because ReactionDrawerItem is using the same prop. It does not have a separate props.

Reply:
We pass the reactions in as a custom component, the DrawerMenuItem doesn't use anything reaction so shouldn't be on its interface. We can create a `interface _ReactionDrawerMenuItemProps extends _DrawerMenuItemProps { reactionsRelatedProps }`

Reply:
(even then _ReactionDrawerMenuItemProps doesn't need to extend the _DrawerMenuItemProps, it can just have its own interface for whatever props it needs)

Reply:
Yes. We can remove them. Thanks :)